internal/characters/lisa/abil.go

```diff
@@ -227,9 +227,11 @@ func (c *char) Burst(p map[string]int) (int, int) {
 	//[8:11 PM] gimmeabreak: i guess single target it does nothing then?
 	//[8:12 PM] ArchedNosi | Lisa Unleashed: yeah single does nothing
 
+	//burst cd starts 52 frames after executed
+	//energy consumed the same time as the initial hit (64 frames)
 	c.ConsumeEnergy(64)
 	// c.CD[def.BurstCD] = c.Core.F + 1200
-	c.SetCDWithDelay(core.ActionBurst, 1200, 64)
+	c.SetCDWithDelay(core.ActionBurst, 1200, 52)
```

Comment:
Do you have a source for this number? Video by any chance?

Reply:
I did my own recording for her burst against ruin guard, want me to share the vid?

But based on the recording:
Burst | Frames
-- | --
Cast | 330
CD Start | 382
Initial Dmg | 394

Reply:
All good. Shizuka confirmed as well :) Thanks so much